Sources/BranchSDK/BranchOpenRequest.m

```diff
@@ -151,6 +152,10 @@ - (void)processResponse:(BNCServerResponse *)response error:(NSError *)error {
             }
         }
     }
+    
+    if (referringURL.length > 0) {
```

Comment:
Should we instead use the function `isLinkingRelatedRequest`?

Reply:
That function is not availbale here. Also we dont store post params inside this object.